public/assets/css/bookmarklet.css

```diff
@@ -57,20 +82,34 @@ h5:first-child {
   background-position: center;
   background-size: cover;
   border: 1px solid #ccc;
+  display: inline-block;
   height: 150px;
   width: 150px;
 }
 
-.collabosphere-item input {
-  margin-top: 2px;
+.collabosphere-item-container label {
+  cursor: pointer;
+}
+
+.collabosphere-items-empty {
+  margin-bottom: 25px;
+}
+
+.collabosphere-items-list:empty + .collabosphere-items-empty {
+  display: block !important;
```

Comment:
I thought I needed it to override the 'hide' class, but I found a better approach and got rid of the `!important`